### PR TITLE
Bc1 24 benefits section show land

### PR DESCRIPTION
**Pull request by sadiGFG**

### CODE REVIEW

**Comment by abdolmajed3** [comment]
please resolve the conflicts

File: show-land/sass/layout/_benefits.scss [hunk 1]
```diff
@@ -0,0 +1,60 @@
+@use "sass:list";
+.benefits {
+  padding-top: list.nth($spaces, 16);
+  padding-bottom: 8rem;
```
Comment (abdolmajed3) [inline show-land/sass/layout/_benefits.scss:4]: i think you can use the dynamic spacing for padding, margin or gaps.
like this :
`list.nth($spaces, 16)`

File: show-land/sass/layout/_benefits.scss [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+    font-size: $fontSizeXl;
+    margin-bottom: list.nth($spaces, 12);
+    text-align: center;
+  }
+
+  &__features {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    row-gap: 4rem;
+    column-gap: 4rem;
```
Comment (abdolmajed3) [inline show-land/sass/layout/_benefits.scss:24]: i think you can use the dynamic spacing for padding, margin or gaps.
like this :
`list.nth($spaces, 16)`

File: show-land/sass/layout/_benefits.scss [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  &__img_box_1 {
+    padding-left: list.nth($spaces, 4);
+    border-radius: $roundedLg;
+    margin-bottom: list.nth($spaces, 4);
+  }
+
+  &__features_1 {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    column-gap: 4rem;
```
Comment (abdolmajed3) [inline show-land/sass/layout/_benefits.scss:57]: it's can be replaced like this:
`list.nth($spaces, 16)`

File: show-land/sass/layout/_home-page.scss [hunk 4]
```diff
@@ -1,5 +1,6 @@
 @use "sass:list";
 
+
 .home-page {
   padding-top: 5rem;
   padding-bottom: 5rem;
```
Comment (abdolmajed3) [inline show-land/sass/layout/_home-page.scss:6]: use variables bro ;)

**Review by abdolmajed3 — APPROVED** [review]

**Review by MaxRichard2007 — APPROVED** [review]

**Pull request merged**